requirements/dev.txt

```diff
@@ -85,9 +85,9 @@ coverage[toml]==7.4.1 \
     # via
     #   -r requirements/dev.in
     #   pytest-cov
-dill==0.3.7 \
-    --hash=sha256:76b122c08ef4ce2eedcd4d1abd8e641114bfc6c2867f49f3c41facf65bf19f5e \
-    --hash=sha256:cc1c8b182eb3013e24bd475ff2e9295af86c1a38eb1aff128dac8962a9ce3c03
+dill==0.3.8 \
```

Comment:
changes in requirements/ need to be collected/squashed onto a single commit, lest madness ensue...

Reply:
I don't know how to do this.